### DISCUSSION TITLE
Run it on no network mode on WSL

### DISCUSSION THREAD

**Discussion by shreya-dama-Crafty-Apes-VFX** [AbdBarho/stable-diffusion-webui-docker#366] [Q&A]
I am trying to build an image with all the required .ckpt files and dependencies to run it on a machine with no network access. Could anybody please help understand what are the required changes and steps to make it possible. Thank you

**Comment by mckinlec** [comment]
I'm interested in the same

**Comment by AbdBarho** [comment]
Download and run the container somewhere where you have internet, open the UI, click all the buttons that you might want to click (which my trigger downloads in the background), and then you can copy the entire folder to another machine.

I know this is not a nice solution, but due to limitations of how these UIs work, there is no easy way.
